documentation/cookbook.qmd

```diff
@@ -0,0 +1,266 @@
+# Kaggle Benchmarks Cookbook
+
+Welcome to the Kaggle Benchmarks Cookbook! This guide provides a collection of "recipes" — practical examples and patterns to help you get the most out of the library.
+
+For a comprehensive overview, refer to the [Quick Start](quick_start.md) and [User Guide](user_guide.md). For starter, please take a look at this [example notebook](https://www.kaggle.com/code/nicholaskanggoog/starter-notebook-template-nick).
+
+## Basics
+
+### Recipe: Defining a Simple Pass/Fail Task
+The most fundamental task type is one that asserts a condition and returns nothing. If all assertions pass, the task succeeds; otherwise, it fails. This is perfect for simple Q&A checks.
+
+```python
+import kaggle_benchmarks as kbench
+
+@kbench.task(name="check_capital")
+def check_capital(llm):
+    # 1. Prompt the model
+    response = llm.prompt("What is the capital of France?")
+
+    # 2. Assert the correctness of the response
+    # We use a regex for robust, case-insensitive matching.
```

Comment:
nit - might be nice to mention there are other assertion mechanisms possible as this is the 'first' example in the cookbook. Just a comment is fine.

Reply:
Done!